docs/source/optimization/params.rst

```diff
@@ -0,0 +1,91 @@
+.. _params:
+
+====================
+The params DataFrame
+====================
+
+The ``params`` DataFrame is an important concept in estimagic. It collects
+information on the dimensionality of the optimization problem, lower and upper
+bounds, categories of parameters and valid ranges for randomly sampled parameter
+vectors. Moreover, it is the mandatory first argument of any criterion function
+optimized with estimagic.
+
+
+If you haven't done so yet, you should check out our example notebook for an
+ordered logit model, so you see one small params DataFrame in action.
```

Comment:
add a reference so the reader can get to the example with one click

Reply:
Had to find out how that works, but now it's done. Also in other places of the documentation.